python/pyproject.toml

```diff
@@ -0,0 +1,67 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools", "wheel"]
+
+[project]
+authors = [
+  {name = "Cucumber Limited", email = "cukes@googlegroups.com"}
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Framework :: Gherkin",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: POSIX",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: MacOS :: MacOS X",
+  "Topic :: Utilities",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11"
+]
+dependencies = [
+  "importlib_resources",
+  "pydantic>=2.0.3"
+]
+description = "Cucumber Messages is a message protocol for representing results and other information from Cucumber. "
+license = {text = "MIT"}
+maintainers = [
+  {name = "Konstantin Goloveshko", email = "kostya.goloveshko@gmail.com"}
+]
+name = "messages"
+readme = {file = "README.md", content-type = "text/markdown"}
+requires-python = ">=3.8"
+urls = {Repository = "https://github.com/cucumber/messages"}
+version = "0.1.0"
```

Comment:
This will need editing to conform to polyglot-release

Reply:
Could you please give a link where I could find how to do this right

Reply:
I don't think we directly have a link. But we use a tool called `polyglot-release` to release internal tools.

If you look at how it is used. You'll see you'll need a version file that is directly auto-edited.

https://github.com/cucumber/polyglot-release

An example of "how" it auto-worked from the CLI can be viewed in this commit for gherkin
https://github.com/cucumber/gherkin/commit/650109d73b70e83fe6c1eb400005084fd5a2f59d

Here is "what" I did.

Ran a command on the CLI, then waited a few seconds or so.
It auto-generated this commit and changed all the relevant files (Check the python change)

Reply:
@luke-hill this currently looks good. Do you see a specific problem?

Reply:
@luke-hill I did both solutions: copying cck to this repo directly and getting them from the repo at the test runtime. @mpkorstanje has asked to not depend on the cck repo itself, so there is just a small amount of tests

Reply:
@mpkorstanje this version should be 25+ not 0.1 - Although maybe we keep them async until we're happy it's stable and bump

Reply:
I have added a release GitHub flow. If I understood correctly, would the version be up for release?

Reply:
@luke-hill then the current version is fine. `polyglot-release` will replace the version number before release.

@elchupanebrej the first release with python in it will probably be a v26.x release. All languages are released in tandem.